src/util/subjects.ts

```diff
@@ -1,5 +1,12 @@
-import { constants } from '@ndla/ui';
+/**
+ * Copyright (c) 2023-present, NDLA.
```

Comment:
Tipper denne egentlig er eldre, men det var det eg fant via vscode.
```suggestion
 * Copyright (c) 2022-present, NDLA.
```